scripts/postCss.mjs

```diff
@@ -1,9 +1,9 @@
-const autoprefixer = require('autoprefixer');
-const postcss = require('postcss');
-const comments = require('postcss-discard-comments');
-const fs = require('fs');
+import autoprefixer from 'autoprefixer';
+import postcss from 'postcss';
+import comments from 'postcss-discard-comments';
+import fs from 'fs';
 
-fs.readFile('src/auro-radio.css', (err, css) => {
+fs.readFile('src/style.css', (err, css) => {
```

Comment:
The contents of the file need to be updated to read multiple css files. See [auro-flight scripts/postCss.mjs](https://github.com/AlaskaAirlines/auro-flight/pull/108/files#diff-fde62f0ca437c7f25bd58615f6f5ee8a06ed36bce9aaebe2c0761a31dfea249f) for how to handle this issue.

Reply:
This still does not work because there are multiple scss files. See https://github.com/AlaskaAirlines/auro-checkbox/blob/master/scripts/postCss.mjs for handling multiple scss files